validator/reports.py

```diff
@@ -138,7 +138,9 @@ def report(self, errors):
                 for path in error.files:
                     self.log.append('\t%s' % str(path))
             if isinstance(error, MdDiff):
-                self.log.append('Files are different:\n\t%s\n\t%s\n\n' % (str(error.base), str(error.other)))
+                self.log.append('Files are different:\n\t%s\n\t%s\n\n' % (str(error.base_path), str(error.other_path)))
```

Comment:
When I tested this, `base_path`  and `other_path ` wasn't a real field on error. Please double check.

Reply:
This confused me, because I didn't purposely change this line. You are correct, I must have accidentally ran a replace when I changed my own variable name, and it got caught in it. Since I didn't test MdDiff, I didn't catch it :/